app/pastes/search-paste/searchPasteValidator.js

```diff
@@ -0,0 +1,33 @@
+class SearchPasteValidator {
+    validate({id, name, authorId}) {
+        const errors = [];
+
+        if (!id && !name && !authorId) {
+            errors.push('Parameters are missing');
+        }
```

Comment:
It actually lets us invoke requests with `{id: 'id-1', name: undefined, authorId: undefined}`, which is forbidden, i think?)